internal/inspectimage/bom_display.go

```diff
@@ -19,12 +19,16 @@ type BOMEntryDisplay struct {
 	Version   string                 `toml:"version,omitempty" json:"version,omitempty" yaml:"version,omitempty"`
 	Metadata  map[string]interface{} `toml:"metadata" json:"metadata" yaml:"metadata"`
 	Buildpack dist.ModuleRef         `json:"buildpacks" yaml:"buildpacks" toml:"buildpacks"`
+	Extension dist.ModuleRef         `json:"extensions" yaml:"extensions" toml:"extensions"`
```

Comment:
I had the occasion to go back and look at this PR, and actually I don't think this is what we want. A "bom entry" is an outdated concept that only applies to buildpacks. It appears that we're basically just repeating information but under the header "extensions"... (cc @jjbustamante )